### PR TITLE
Find and Link Trophy Room - Change page to UNDER CONSTRUCTION temporarily 

### PR DESCRIPTION
**Pull request by artofmayhem**
This PR is meant to fix the non working status of the trophy room button on /gamplay path

In looking through the code I found an iteration of a trophy page with comprehensive logic in it not tied to anything. I went ahead and linked it to the Button adding an event handler to push the user to the props.baseURL/gameplay/trophy-room and called that function on click utilizing the existing button element contained in GamePlayMain.js. I created a route utilizing the same switch statement space that the gameplay mission route was in along with the 404 not found message, and GameModeBtns. 

After getting the existing RenderTrophyRoom component to be visible i was able to view the rendered elements that were not tied to data yet.  As I unsure of the future usage of the existing code, I commented out all that was not needed and replaced it with TEMPORARY labels noting what each was for. Prior commented out .maps waiting for data and imports that were unused were labeled as such. 

Each area that was adjusted was labeled for future devs to finish. 

I also added some temporary styling to make the new under construction elements match the figma design. To ensure that any future user did not get stuck in the under construction page I created a button matching the style of the rest of the page and utilizing the atma font. 

This page is now functioning as currently intended with clean navigation to and from. 

RERFERENCING  Trello Card: https://trello.com/c/NpuFouxI

PR VIDEO: https://youtu.be/x43RFktKxmU

### CODE REVIEW

File: src/components/pages/GamePlay/GamePlayMain.js [hunk 1]
```diff
@@ -96,7 +108,9 @@ const GamemodeBtns = props => {
           Accept The Mission
         </button>
 
-        <button>Trophy Room</button>
+        <button className="" onClick={trophyRoom}>
```
Comment (OniWarrior) [inline src/components/pages/GamePlay/GamePlayMain.js]: Your className for your button is empty. Easy fix, just add a class name.

Comment (artofmayhem) [reply src/components/pages/GamePlay/GamePlayMain.js]: no className needed REMOVED

**Review by OniWarrior — DISMISSED** [review]
Other than adding a classname to the empty classname attribute for the trophy room button. Everything checks. I didn't see anything else that needed to be corrected.

Update: I refreshed my page. I saw that you removed the class name. There's nothing else I need to comment on. I approve.

File: src/components/pages/TrophyRoom/RenderTrophyRoom.js [hunk 2]
```diff
@@ -1,77 +1,112 @@
...
-      <Header displayMenu={true} title="STORY SQUAD" />
-      <Button
-        style={{ margin: '1rem' }}
-        className="back-btn"
-        onClick={dashboard}
-      >
-        Back to Child Dashboard
-      </Button>
+      {/* TEMPORARY  BELOW SECTION IS THE UNDER CONSTRUCTION SECTION TO BE REMOVED WHEN FUNCTIONAL PAGE IS READY TO BE USED*/}
+      <div className={'under-construction'}>
```
Comment (Mitch98k) [inline src/components/pages/TrophyRoom/RenderTrophyRoom.js:79]: If I'm guessing correctly, the brackets around the classnames here are meant to indicate that this is temporary and will be removed later and to distinguish them from ones that won't be removed?

File: src/components/pages/TrophyRoom/RenderTrophyRoom.js [hunk 3]
```diff
@@ -123,9 +166,9 @@ const RenderLeaderboard = props => {
           >
             <h3>Skins</h3>
             <h3>Stickers</h3>
-            <h3>Collectibles</h3>
-            {/* Map over inventory state here */}
-          </Modal>
+            <h3>Collectibles</h3> */}
```
Comment (Mitch98k) [inline src/components/pages/TrophyRoom/RenderTrophyRoom.js:169]: Is the closing comment bracket here just the end of the opening one on line 157, and everything between that is commented out? That's what I figure, but if not I don't know why it is there.

File: src/styles/less/TrophyRoom.less [hunk 4]
```diff
@@ -197,8 +197,8 @@
   text-align: center;
 }
 .ant-modal-body p {
-  color: #939AFF;
-  text-shadow: -4px 0 #FFFFFF, 0 4px #FFFFFF, 4px 0 #FFFFFF, 0 -4px #FFFFFF;
+  color: #939aff;
+  text-shadow: -4px 0 #ffffff, 0 4px #ffffff, 4px 0 #ffffff, 0 -4px #ffffff;
```
Comment (Mitch98k) [inline src/styles/less/TrophyRoom.less:201]: Nice job keeping the casing consistent with these changes. Changing to lowercase makes reading this code a lot easier in the eyes I think.

**Review by Mitch98k — APPROVED** [review]

File: src/components/pages/TrophyRoom/RenderTrophyRoom.js (hunk 3 above)
Comment (artofmayhem) [reply src/components/pages/TrophyRoom/RenderTrophyRoom.js:169]: it's to bracket in what needs to be removed once the regular rendering is in place to that it is pretty clear i put little up arrows to point at the section as well

**Review by rhiannonstanford — APPROVED** [review]
Excellent attention to detail.  This PR will help begin the audit on routes and ensuring overall app functionality.  Thank you!

**Pull request merged**